docs/signaling.md

```diff
@@ -2,17 +2,55 @@
 
 P2P layer. Handles sending and receiving messages through TomP2P.
 
-You need an instance of [ClientService]. Through this, you will do all your operations.
+An instance of [ClientService] is needed. All operations are done through this.
 
-Specifically, you can [add a new client][ClientService.addClient]. This will get you an [InternalClient]. You then add a message listener to this client, which will relay the messages you receive via the TomP2P layer.
+Specifically, one can [add a new client][ClientService.addClient]. This will return an [InternalClient]. A message listener can then be added to this client, which will relay the messages the client receives via the TomP2P layer.
 
-To connect to other clients, you [find clients][ClientService.findClient]. This will get you an [ExternalClient]. You can then use an InternalClient to [send messages][InternalClient.sendMessage] to this ExternalClient. (This will call the message listener on the other peer's InternalClient you're sending messages to.)
+To connect to other clients, [find clients][ClientService.findClient]. This will return an [ExternalClient]. An [InternalClient] can then be used to [send messages][InternalClient.sendMessage] to this ExternalClient. (This will call the message listener on the other peer's InternalClient).
 
 ## Design decisions
 
 ### Anti-cheating mechanisms
 
-As we're in a P2P network, it is possible to add bad data. However, we wanted to make this a bit harder, so we at least don't expose it on our API.
+As this system in a P2P network, it is possible to add bad data. However, the API makes this harder:
```

Comment:
As I think about it. I'd change the title and remove this sentence. Let's just say that the stuff get's overwritten.